pkg/handlers/config.go

```diff
@@ -1147,6 +1148,15 @@ func (h *Handler) SetAppConfigValues(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	validationErrors := kotsadmvalidator.ValidateConfigSpec(renderedConfig.Spec)
+	if len(validationErrors) > 0 {
+		setAppConfigValuesResponse.Error = "failed to validate config values"
+		setAppConfigValuesResponse.ValidationErrors = validationErrors
+		logger.Error(errors.Wrap(err, setAppConfigValuesResponse.Error))
```

Comment:
Same as my other comment.  The `renderedConfig` above this creates a new error to log instead of wrapping.

Reply:
yeah, I should have logged the `setAppConfigValuesResponse.Error`